alembic/versions/c0e732b29384_initial_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Initial migration
+
+Revision ID: c0e732b29384
+Revises: 248d928a8bc4
+Create Date: 2024-10-31 11:44:09.714102
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'c0e732b29384'
+down_revision: Union[str, None] = '248d928a8bc4'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    pass
```

Comment:
The `downgrade` function is also empty. Ensure that this is intentional. If you need to revert changes made in the `upgrade` function, you should add the necessary operations here.